code/orders-api-rest/api/openapi-rest.yml

```
@@ -816,48 +816,179 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /v1/password-reset/{token}:
+    get:
+      tags:
+        - PasswordResetController
+      summary: Validate password reset token
+      description: |
+        Validates a token and returns its status. Used to enable the password 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)